src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -260,7 +260,7 @@ export function TransactionErrorContent({
   confirmText?: string
 }) {
   const theme = useTheme()
-  const [showDetail, setShowDetail] = useState<boolean>(true)
+  const [showDetail, setShowDetail] = useState<boolean>(false)
```

Comment:
why do we hide it?

Reply:
We should extract minimal message for user, then if they want to see more, they can expand the raw message. I have improved the friendlyError so I think we dont need to show the raw message on default anymore.